### PR TITLE
Re-enable developer setup

### PR DESCRIPTION
**Pull request by jimbo**
The original method of setting up a development environment is gone. Here, I've included all the changes I needed to get it working again. Also included:

- Mock GraphQL (by @zetlen)
- Rename from `rush` to `venia`

### CODE REVIEW

File: theme/devtools/devserver-graphql-simulator.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const { makeExecutableSchema, addMockFunctionsToSchema } = require('graphql-tools');
```
Comment (DrewML) [inline theme/devtools/devserver-graphql-simulator.js]: Doesn't seem like this file is getting picked up by `prettier` or `eslint`. Can we check why that is?

File: theme/webpack.config.js [hunk 2]
```diff
@@ -1,76 +1,62 @@
...
 const UglifyPlugin = require('uglifyjs-webpack-plugin');
-const WorkboxPlugin = require('@magento/workbox-webpack-plugin');
-const WriteFileWebpackPlugin = require('write-file-webpack-plugin');
 const configureBabel = require('./babel.config.js');
-const getMagentoEnv = require('./devtools/get-magento-env');
-const express = require('express');
-const {
-    WebpackMagentoRootComponentsChunksPlugin
-} = require('@magento/anhinga');
+const { Webpack: WebpackTools } = require('@jzetlen/pwa-buildpack');
```
Comment (DrewML) [inline theme/webpack.config.js]: Believe Zetlen merged this

Comment (zetlen) [reply theme/webpack.config.js]: Yeah, that was my bad. I pushed an update directly to master to fix it during Imagine.

File: theme/devtools/devserver-graphql-simulator.js (hunk 1 above)
Comment (jimbo) [reply theme/devtools/devserver-graphql-simulator.js]: @DrewML I'll check. But as I noted, files in `devtools/` are marked for deletion anyway.

File: theme/webpack.config.js [hunk 3]
```diff
@@ -105,91 +91,84 @@ module.exports = async env => {
...
+    } else {
+        config.devServer = await PWADevServer.configure({
+            publicPath,
+            backendDomain,
+            serviceWorkerFileName,
+            paths,
+            id: 'magento-venia'
+        });
+
+        if (process.env.MOCK_GRAPHQL) {
```
Comment (DrewML) [inline theme/webpack.config.js]: Can we pull this out into a separate PR? Have some questions/concerns that shouldn't block the rest of this PR from landing

Comment (jimbo) [reply theme/webpack.config.js]: @zetlen

Comment (zetlen) [reply theme/webpack.config.js]: Yes, please feel free to delete.

**Comment by zetlen** [comment]
@jimbo Can you update `package-lock.json` on this PR to pull in the latest version of buildpack?

**Comment by DrewML** [comment]
> @jimbo Can you update package-lock.json on this PR to pull in the latest version of buildpack?

For posterity's sake, done in #55.

**Comment by DrewML** [comment]
Looks like the majority of the conflicts with `master` right now are just due to the rename in #49. Hopefully an easy fix.

**Comment by zetlen** [comment]
No longer necessary because of other merges. 🎉
